src/problems/tsp/tsp.h

```diff
@@ -34,6 +34,8 @@ class TSP : public VRP {
   Matrix<Cost> _symmetrized_matrix;
   bool _round_trip;
 
+  Cost compute_cost(const std::list<Index>& tour, const Matrix<Cost>& matrix) const;
```

Comment:
It looks like `compute_cost` does not even need to be a member function since I don't think it uses any other member from `TSP`. We could avoid exposing it altogether in the header and simply having it lie in the `.cpp` file as a non-member.